ERCS/erc-7825.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 7825
+title: State Channel Standard data types and interfaces
+description: Interoperable state channels on Ethereum using State Channel Protocol, integrated with ERC-4337 for enhanced account abstraction.
+author: State Channels (@statechannels), Layer-3 Foundation (@layer-3), Louis Bellet (@mod)
+discussions-to: https://ethereum-magicians.org/c/ercs/57
```

Comment:
Please create a discussions topic in Eth Magicians linking to this PR
https://ethereum-magicians.org/c/ercs/57